website/docs/r/availability_set.html.markdown

```diff
@@ -42,8 +42,12 @@ The following arguments are supported:
 
 * `platform_update_domain_count` - (Optional) Specifies the number of update domains that are used. Defaults to 5.
 
+~> **NOTE:** The number of Update Domains varies depending on which Azure Region you're using - [a list can be found here](https://github.com/MicrosoftDocs/azure-docs/blob/04809e3072e0061ca436b0986cf2222c472651ba/includes/managed-disks-common-fault-domain-region-list.md).
```

Comment:
Can we refer to the master branch instead?
https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/managed-disks-common-fault-domain-region-list.md

Reply:
So the reason not to do that is in case the file gets renamed / removed, but yeah as you’ve mentioned this may make sense too. Given it’s available in the git history, we could probably update this however

Reply:
If the file happens to disappear, then an update would make sense. Using master IMHO prevents this doc from being updated as often, when new regions appear or get supported.

Reply:
fixed :)